shred uses the rdtsc instruction, which will not compile on any 
machine less than i586.  i486 is the only architecture supported 
by gentoo affected.

Fix by taviso@gentoo.org

http://bugs.gentoo.org/show_bug.cgi?id=32429

CVS shred.c has been rewritten so it doesn't contain this asm 
code anymore ... which means it's indirectly fixed ;)

--- coreutils-5.0.91/src/shred.c
+++ coreutils-5.0.91/src/shred.c
@@ -518,7 +518,7 @@
 #define ISAAC_SEED(s,x) isaac_seed_data (s, &(x), sizeof (x))
 
 
-#if __GNUC__ >= 2 && (__i386__ || __alpha__)
+#if __GNUC__ >= 2 && (__i386__ || __alpha__) && !(__i486__)
 /*
  * Many processors have very-high-resolution timer registers,
  * The timer registers can be made inaccessible, so we have to deal with the
